src/ucp/api/ucp.h

```diff
@@ -2235,16 +2235,16 @@ ucs_status_t ucp_worker_signal(ucp_worker_h worker);
 
 /**
  * @ingroup UCP_WORKER
- * @brief Accept connections on a local address of the worker object.
+ * @brief Create a listener to accept connections on. Connection requests on
+ * the listener will arrive on a local address specified by the user.
  *
- * This routine binds the worker object to a @ref ucs_sock_addr_t sockaddr
+ * This routine binds the listener object to a @ref ucs_sock_addr_t sockaddr
  * which is set by the user.
- * The worker will listen to incoming connection requests and upon receiving such
- * a request from the remote peer, an endpoint to it will be created.
- * The user's call-back will be invoked once the endpoint is created.
+ * The listener will listen to incoming connection requests and upon receiving
+ * such a request from the remote peer, an endpoint to it will be created.
```

Comment:
"an endpoint to it will be created." is not accurate, user can create endpoint later on from conn_req

Reply:
didn't mean that the ep would necessarily be created in this function. but ok, i see your point, will clarify.

Reply:
how about:
"The listener will listen to incoming connection requests. After receiving a request from the remote peer, an endpoint to it will be created - either right away or later on, depending on the user's setting in @_ref ucp_listener_params_t"

Reply:
"later on" -> `by calling @ref ucp_ep_create, as specified by the callback type in @ref ucp_listener_params_t`